common-services.yml

```diff
@@ -50,4 +48,26 @@ services:
         max-buffer-size: 100m
       driver: json-file
 
+  exporter_postgres:
+    image: audius/exporter-postgres:v0.10.1
+    restart: always
+
+  exporter_elasticsearch:
+    image: audius/exporter-elasticsearch:v1.3.0
+    restart: always
+
+  exporter_redis:
+    image: audius/exporter-redis:v1.44.0-alpine
+    restart: always
 
+  exporter_linux:
+    image: quay.io/prometheus/node-exporter:v1.4.0
```

Comment:
how come this isn't one we build? does it matter?

Reply:
We could build it and push it ourselves so we know we're locking the image down to a timestamp, but the others required custom code to be able to use prod.env and overrides.env variables.

Since this one didn't require that custom code (since it doesn't consume our variables), I figured just to use the official image since our image would be a one-liner with `FROM quay.io/prometheus/node-exporter:v1.4.0`.

I'm +0 on either approach. Thoughts?

Reply:
ya, probs fine. no need to change. helpful ctx